x-pack/plugins/security_solution/server/lib/entity_analytics/risk_score/calculate_risk_scores.ts

```diff
@@ -235,7 +236,9 @@ export const calculateRiskScores = async ({
     const scriptedMetricPainless = await getPainlessScripts();
     const filter = [
       filterFromRange(range),
-      { bool: { must_not: { term: { [ALERT_WORKFLOW_STATUS]: 'closed' } } } },
+      ...(excludeAlertStatuses.length > 0
```

Comment:
Nit: I think extracting the ternary usually helps with readability

Reply:
Agreed!